src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { getUserById } from '../../services/userService';
+import { Todo } from '../../types/todo';
+import { User } from '../../types/user';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user: User | null = getUserById(todo.userId);
```

Comment:
Ensure that the `getUserById` function can handle cases where a user might not be found and returns `null` or a default user object. This will prevent potential rendering issues in the `UserInfo` component when `user` is `null`.